gradle/build-gradle-plugins.gradle

```diff
@@ -23,8 +23,11 @@ propertiesFile.withInputStream {
 	versions.load(it)
 }
 
+//apply from: 'publish-maven.gradle'
+//task install(dependsOn:'publishToMavenLocal'){}
```

Comment:
I this meant to be commented out?
Seems like it's in `publish-maven.gradle` now?